docs/rules/attribute-names.md

```diff
@@ -4,7 +4,10 @@ Attributes are always treated lowercase, but it is common to have camelCase
 property names. In these situations, an explicit lowercase attribute should
 be supplied.
 
-Further, camelCase names should ideally be exposed as snake-case attributes.
+Further, camelCase names should ideally be exposed as dash-case attributes.
```

Comment:
we should probably call it kebab-case, the same as MDN:
https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case